ibmcloud/README.md

```diff
@@ -77,136 +91,65 @@ kubectl label node $node node-role.kubernetes.io/worker=
 ```
 
 ## Deploy the Confidential-containers operator
+The `caa-provisioner-cli` simplifies deploying the operator and the cloud-api-adaptor resources on to any cluster. See the [test/tools/README.md](../test/tools/README.md) for full instructions. To create an ibmcloud ready version follow these steps
+
+```
+cd test/tools
```

Comment:
I wonder if using a pushd/popd combo here might be better as the next set of docs seems to assume we are in the repo root directory again?